runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -269,6 +309,8 @@ public XWalkViewInternal(Context context) {
     @XWalkAPI(preWrapperLines = {
                   "        super(${param1}, ${param2});",
                   "        if (isInEditMode()) return;",
+                  "        mAnimatable = ${param2}.getAttributeValue(",
+                  "                XWALK_ATTRS_NAMESPACE, \"animatable\");",
```

Comment:
Why not just use the string directly for the name space? Or use the static member for both of name space and attribute name.